requirements/edx/base.txt

```diff
@@ -115,8 +115,8 @@ edx-completion==0.1.7
 edx-django-oauth2-provider==1.2.5
 edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
-edx-drf-extensions==1.4.0
-edx-enterprise==0.69.4
+edx-drf-extensions==1.5.1
```

Comment:
@Squirrel18 this other dependencies were changed automatically, correct?
@jmbowman should this be left here, or is it better to remove anything not related to `modernize`?

Reply:
We're now defaulting to upgrading dependencies that have new releases available, so they don't fall really far behind.  It's fine to upgrade them as long as tests still pass and there aren't any obviously problematic breakages of backwards compatibility in the changelogs of the upgraded packages.

Reply:
That is good to know.